docusaurus.config.js

```diff
@@ -35,18 +35,17 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          lastVersion: "2.0",
+          lastVersion: "current",
           versions: {
             current: {
-              label: "old",
-              path: "old",
+              label: "stable",
               badge: false,
-              banner: "unmaintained",
             },
-            "2.0": {
+            "1.0": {
```

Comment:
might be worth renaming this "old" just to match - doesn't have to be a number string afaik